SimPEG/electromagnetics/natural_source/receivers.py

```diff
@@ -29,318 +31,77 @@ class BaseRxNSEM_Point(BaseRx):
         {
             "real": ["re", "in-phase", "in phase"],
             "imag": ["imaginary", "im", "out-of-phase", "out of phase"],
+            "apparent resistivity": [
```

Comment:
Should this be on the base? or should we just over-write `component` for the complex resistivity receiver? I am inclined to do the latter since for example `apparent resistivity` isn't a valid `component` choice for the impedance receiver.

Reply:
make sense to me! 👍